src/index.js

```diff
@@ -5,19 +5,32 @@ function startsWith (key, val) {
   return key.indexOf(val) === 0;
 }
 
+function isAttribute (key) {
+  return key === 'attributes';
+}
+
 function shouldBeAttr (key, val) {
-  return startsWith(key, 'aria-') || startsWith(key, 'data-');
+  return startsWith(key, 'aria-') || startsWith(key, 'data-') || isAttribute(key);
 }
 
 function handleFunction (Fn) {
   return Fn.prototype instanceof HTMLElement ? new Fn() : Fn();
 }
 
+function setAttr (node, attrName, attrValue) {
```

Comment:
It's calling `node.setAttribute()` if `attributes` is not defined. What I'm saying is that it should be setting a prop if no `attributes` is defined.

> I agree, but that should be addressed in next PR

Sure, but it'd be nice if it could happen soon after.